data/campaigns/tutorial/scenarios/01_Tutorial_part_1.cfg

```diff
@@ -617,6 +723,13 @@
                 message= _"Be sure to examine the <i>traits</i> of your new recruits. They are listed under its race in the sidebar. Traits can subtly affect how you use your troops. For example, units with the <i>quick</i> trait can move a extra hex each turn, and units with the <i>intelligent</i> trait require 20% less experience to level up."
             [/message]
 
+            # Allow our hero to move freely until next turn without undo messages now that the recruiting is done
+            [store_locations]
+                variable=target_loc
+                x,y=9,6
+                radius=4
+            [/store_locations]
+
```

Comment:
Is the intention here to never show the undo message?
In that case, leaving out the filtering may match the whole map.

Reply:
Yes, that was the intention. If we do so as you hinted, we would calculate distances for all over the map which is unnecessary - much less CPU intensive to check distance of a bunch of hexes around our hero than the entire map. (I don't know the engine so maybe I'm wrong.)

The more direct way would be to add an enable-disable variable and check that first in the check_undo moveto event. But that needs another variable, another level of ifs and a similar amount of code lines as I see. Should I change to this instead?